src/services/sdk.ts

```diff
@@ -244,28 +239,73 @@ const handleRegistryUpdate = (dispatch: Dispatch) => (
   );
 };
 
+export const isGraphChangeAnnouncement = (
+  obj: unknown
+): obj is GraphChangeAnnouncement => {
+  return (
+    (obj as Record<string, unknown>)["announcementType"] ===
+    AnnouncementType.GraphChange
+  );
+};
+
+export const isBroadcastAnnouncement = (
+  obj: unknown
+): obj is BroadcastAnnouncement => {
+  const type = (obj as Record<string, unknown>)["announcementType"];
+  return (
+    type === AnnouncementType.Profile ||
```

Comment:
```
[ 
  AnnouncementType.Profile,
  AnnouncementType.Broadcast,
  AnnouncementType.Reply
 ].some( (item) => item === type)
```

maybe another way to write this

Reply:
This is nice, but the way it is a a little simpler (a little dumber), so I think I'll keep it.